cloudify_cli/commands/deployments.py

```diff
@@ -227,10 +231,16 @@ def manager_create(blueprint_id,
             skip_plugins_validation=skip_plugins_validation
         )
     except MissingRequiredDeploymentInputError as e:
-        logger.info('Unable to create deployment. Not all '
-                    'required inputs have been specified...')
-        _print_deployment_inputs(client, blueprint_id)
-        raise CloudifyCliError(str(e))
+        expected_inputs_def = _get_deployment_inputs(client, blueprint_id)
+        available_input_names = set(inputs.keys())
```

Comment:
And in any case, this logic should be done in the DSL parser, or on the server side, anyway. We're trying to move the CLI towards being a rather stupid conduit for REST commands.

Reply:
Surely we still need to handle missing input stuff on the client side anyway, for cfy local cases if not to speed up returning of useful errors to a user rather than requiring a roundtrip to the manager?

Reply:
@mcouthon @geokala all the info required to get these details, is already available on the manager side.
At the time of throwing the exception on the manager side, we have all the info. Unfortunately, our Exception classes are "flat" and don't contain fine-grained fields, so we can't pass this info back from the manager to the CLI in an easy way (all we have at the CLI side is just a flat string explaining what's wrong).

Alternatively what we could do is simply print out the error message received from the REST service and remove `_print_deployment_inputs` altogether.